runbotics/runbotics-scheduler/src/scheduler-database/credential-collection-user/credential-collection-user.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { UserEntity } from '#/database/user/user.entity';
+import {
+    Column,
+    Entity,
+    JoinColumn,
+    ManyToOne,
+    PrimaryGeneratedColumn,
+} from 'typeorm';
+import { CredentialCollection } from '../credential-collection/credential-collection.entity';
+
+export enum PrivilegeType {
+    READ = 'READ',
+    WRITE = 'WRITE',
+}
+
+@Entity({
+    name: 'credential_collection_user',
+    schema: 'scheduler',
+})
+export class CredentialCollectionUser {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+
+    @ManyToOne(() => CredentialCollection, (collection) => collection, { onDelete: 'CASCADE' })
+    @JoinColumn({ name: 'collection_id' })
+    credentialCollection: CredentialCollection;
+
+    @Column({ name: 'collection_id', type: 'uuid' })
+    credentialCollectionId: string;
+
+    @ManyToOne(() => UserEntity, (user) => user, { onDelete: 'SET NULL' })
+    @JoinColumn({ name: 'user_id' })
```

Comment:
same here, see docs example:
https://typeorm.io/many-to-many-relations#many-to-many-relations-with-custom-properties

I don't think that is necessary, maybe it messed up updating the collection?